edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_variables.scss

```diff
@@ -1,19 +1,21 @@
 // Theme variables.
 
 //color
-$primary: #007FA3;
+$primary: #005A70;
 $inverse: #FFF;
 $secondary: #ffbd3c;
 $gray: #dbdbdb;
 $footer-background: #0d2b52;
 $footer-contrast: rgba(255,255,255,0.7);
 $dark: #313131;
+$gray: #E6E6E6;
```

Comment:
I think this is an override for `$gray` at line 7. Could  we use just one variable? @Squirrel18

Reply:
Addresed. @diegomillan